.pre-commit-config.yaml

```diff
@@ -43,7 +43,13 @@ repos:
     hooks:
     -   id: mypy
 
--   repo: https://github.com/pre-commit/mirrors-pylint
-    rev: v2.3.1
+-   repo: local
     hooks:
     -   id: pylint
+        language: system
+        name: PyLint
+        files: \.py$
+        entry: python3 -m pylint
+        args:
+        - msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
+        - output-format=colorized
```

Comment:
(same here)

Reply:
@schenkh ^

Reply:
@webknjaz what I should do locally if I accepted suggestion for commit here? Pull remote branch?

Reply:
Yes. I usually do `git pull --rebase` (optionally with `--autostash`).

In fact, if you use omz, you should have its Git plugin activated by default and it has a lot of nice aliases like `gup`, `gl`, `gcb`, `gco` and others.
You can check out the whole list by running `alias | grep git`.